ios/Config/Config.xcconfig

```diff
@@ -0,0 +1,19 @@
+//
+//  Config.xcconfig
+//  StarterApp
+//
+//  Created by Mark DeLaVergne on 3/29/23.
+//
+
+// Configuration settings file format documentation can be found at:
+// https://help.apple.com/xcode/#/dev745c5c974
+
+// Versioning
+BUILD_NUMBER=1
```

Comment:
Should this try to read the build number from bitrise?
```BUILD_NUMBER=$(BITRISE_BUILD_NUMBER:default=1)```

Reply:
No - we may (or may not) override it to that in our build process, but we don't want that hard-coded in this repo.